### ISSUE TITLE
Insane TPS lag caused by MoreRed

### ISSUE THREAD

**Issue by ViRb3** [AllTheMods/ATM-6#259]
### Cross-posted at: https://github.com/Commoble/morered/issues/8

I am running [AllTheMods 6](https://www.curseforge.com/minecraft/modpacks/all-the-mods-6) modpack version 1.2.0, which comes with MoreRed installed. My host is PebbleHost Premium, i9 10th gen @ 5GHz with 5GB RAM. When I walk around the map and generate new chunks, I get insane TPS drops - from 20 to about 4-5, and sometimes actions and chat messages will lag for up to 20 seconds. I used the [spark](https://www.curseforge.com/minecraft/mc-mods/spark) mod to profile the server, and it returned this log:
https://spark.lucko.me/#PsDmKOHFeC

If you set the mappings to MCP 1.16.3, and you expand down the chain, you will notice the following lines consuming almost all of the CPU time:

```
net.minecraftforge.eventbus.EventBus.doCastFilter()80.87%65308ms
commoble.morered.MoreRed$$Lambda$5017/1515141381.accept()80.87%
commoble.morered.MoreRed.onPlayerStartWatchingChunk()80.87%
net.minecraft.world.World.getChunkAt()80.83%
```

After I removed MoreRed, the server TPS never goes below 20. I don't know if this is an issue in this mod by itself or a conflict, but it is making the game seriously unplayable.

**Comment by CrankySupertoon** [comment]
Sorry we dont help people reposting our modpack with Forge Haxs, A Pirated Copy of Optifine, and Baritone

**Comment by ViRb3** [comment]
This issue has been confirmed and fixed by the author in `morered-1.16.3-2.0.1.0.jar`, changelog:
```
* Fixed a bug that significantly impacted server performance whenever players began tracking chunks
```
Will keep this issue open until MoreRed is updated in next ATM6 release.
